forms/jqm/elements/subform.js

```diff
@@ -15,7 +15,10 @@ define(function (require) {
     tagName: 'section',
 
     initialize: function(){
-      this.listenTo(this.model, 'update:fieldErrors', this.renderErrors.bind(this) );
+      this.listenTo(this.model, 'update:fieldErrors', function(){
+        this.renderErrors.apply(this, arguments);
+      }.bind(this) );
```

Comment:
the removed line will create a new function bound to the subform. This function will be whatever is being bound *at binding time*

The added lines will run the function found *at run time*, which means that you can override `subFormElement.prototype#renderErrors` or `subFormElementInstance#renderErrors` at any point in time. 

Since we have no idea when a user will override the function, i figured it was safer to go this way